data/src/main/java/com/microsoft/azure/kusto/data/ClientRequestProperties.java

```diff
@@ -25,15 +25,15 @@
  * check out https://docs.microsoft.com/en-us/azure/kusto/api/netfx/request-properties#list-of-clientrequestproperties
  */
 public class ClientRequestProperties {
-    private static final String OPTIONS_KEY = "Options";
-    private static final String PARAMETERS_KEY = "Parameters";
     public static final String OPTION_SERVER_TIMEOUT = "servertimeout";
     public static final String OPTION_CLIENT_REQUEST_ID = "ClientRequestId";
+    public static final Pattern KUSTO_TIME_REGEX =
+            Pattern.compile("(-?)(?:(\\d+)(\\.))?(?:([0-2]?\\d)(:))?([0-5]?\\d)(:)([0-5]?\\d)(?:(\\.)(\\d+))?",
```

Comment:
TIMESPAN

Reply:
I'm naming these for the Kusto datatypes they're relevant to, and this is for the datatype called, "time".

Reply:
Oops, I just checked out https://docs.microsoft.com/en-us/azure/data-explorer/kusto/query/scalar-data-types/ and see that the datatype's name is indeed "timespan". Weird, because I've only ever seen it as "time(...)".

I'll update the terminology accordingly.